libsolidity/codegen/CompilerUtils.h

```diff
@@ -169,8 +169,11 @@ class CompilerUtils
 	/// Stack post:
 	void storeStringData(bytesConstRef _data);
 
-	/// Appends code that cleans higher-order bits for integer types.
-	void cleanHigherOrderBits(IntegerType const& _typeOnStack);
+	/// Helper function for handling fixed point shifting
```

Comment:
Much more important that the fact that it is a helper function: What exactly does it do?